input_raw.go

```diff
@@ -196,14 +198,10 @@ func (i *RAWInput) addStats(mStats tcp.Stats) {
 	i.Unlock()
 }
 
-func startHint(pckt *tcp.Packet) (isIncoming, isOutgoing bool) {
+func http1StartHint(pckt *tcp.Packet) (isIncoming, isOutgoing bool) {
 	isIncoming = proto.HasRequestTitle(pckt.Payload)
 	if isIncoming {
 		return
 	}
 	return false, proto.HasResponseTitle(pckt.Payload)
 }
-
```

Comment:
@code4wt this method was consuming a lot of resources on high traffics. For now, the master branch should not be used to build goreplay because of this issue #859.

Reply:
ok, I saw you wrote a new endhine function in the #863。

Reply:
yes, it was fixed! although it's kinda more complex.